doc/source/language/delegates.rst

```diff
@@ -369,31 +370,31 @@ normally be in :ref:`scope <scope>`.
 Kinds of Delegate (no suffixes)
 ===============================
 
-Under the hood, kOS handles several different kinds of 'functions' and
+Under the hood, kOS handles several different kinds of "functions" and
 methods that aren't actually implemented the same way.  A ``KOSDelegate``
 attempts to hide the details of these differences from the user, but
 one difference in particular still stands out.  In kOS version 0.19.0,
 you cannot reliably make a delegate of a suffix just yet.  (*This is
 intended as a future feature though.  It's been put off because it
-involves decisions that impact the future of the language and once made,
-can't be changed easily.*)
+involves decisions that impact the future of the language and which, once
+made, can't be changed easily.*)
 
 - You **can** make a delegate of a :ref:`user function <user_functions>`
-  implemented in Kerboscript code.::
+  implemented in Kerboscript code::
```

Comment:
The way I try to get around the colon limitation is to put the double colon on the next line and that seems to suppress the compiled html having a colon.  A quick compile test shows that it appears to work in this instance as well:
```
- You **can** make a delegate of a :ref:`user function <user_functions>`
  implemented in Kerboscript code.
  ::
```